### PR TITLE
Complete German translation

### PR DESCRIPTION
**Pull request by zuphilip**
This should complete the translation of the German localization again. The translation is a little free-style because I couldn't think about a good word-by-word translation of the English original text.

### CODE REVIEW

**Comment by zuphilip** [comment]
I rebased this PR to (new) master to re-trigger Travis, which was fixed in the meantime. Now, all tests are passing.

File: locale/de_DE.UTF-8/LC_MESSAGES/palma.po [hunk 1]
```diff
@@ -206,6 +206,17 @@ msgid ""
 "            onclick=\"window.open(this.href); return false;\">MacBook</a>\n"
 "            (RealVNC)."
 msgstr ""
+"Zur Bildschirmfreigabe von Ihrem Laptop benutzt PalMA die VNC Software.\n"
+"            Diese muss einmalig heruntergeladen werden.\n"
+"            Für <a href=\"http://www.bib.uni-mannheim.de/fileadmin/pdf/ub/"
+"LearningCenter/palma-kurzanleitung.pdf\"\n"
+"            onclick=\"window.open(this.href); return false;\">Windows</"
+"a>\n"
+"            gibt es einen vorkonfigurierter UltraVNC-Klient und für\n"
```
Comment (stweil) [inline locale/de_DE.UTF-8/LC_MESSAGES/palma.po]: "Klient" looks strange for me.

File: locale/de_DE.UTF-8/LC_MESSAGES/palma.po [hunk 2]
```diff
@@ -206,6 +206,17 @@ msgid ""
 "            onclick=\"window.open(this.href); return false;\">MacBook</a>\n"
 "            (RealVNC)."
 msgstr ""
+"Zur Bildschirmfreigabe von Ihrem Laptop benutzt PalMA die VNC Software.\n"
+"            Diese muss einmalig heruntergeladen werden.\n"
+"            Für <a href=\"http://www.bib.uni-mannheim.de/fileadmin/pdf/ub/"
```
Comment (stweil) [inline locale/de_DE.UTF-8/LC_MESSAGES/palma.po]: Ups, broken link. They obviously changed their website: ;.)

File: locale/de_DE.UTF-8/LC_MESSAGES/palma.po [hunk 3]
```diff
@@ -206,6 +206,17 @@ msgid ""
...
+"LearningCenter/palma-kurzanleitung.pdf\"\n"
+"            onclick=\"window.open(this.href); return false;\">Windows</"
+"a>\n"
+"            gibt es einen vorkonfigurierter UltraVNC-Klient und für\n"
+"            <a href=\"http://www.bib.uni-mannheim.de/fileadmin/pdf/ub/"
```
Comment (stweil) [inline locale/de_DE.UTF-8/LC_MESSAGES/palma.po]: This link is also broken.

File: locale/de_DE.UTF-8/LC_MESSAGES/palma.po (hunk 1 above)
Comment (zuphilip) [inline locale/de_DE.UTF-8/LC_MESSAGES/palma.po]: Okay, then I suggest:
```
"gibt es eine vorkonfigurierte Software (auf Basis von UltraVNC) und für\"
```
okay?

File: locale/de_DE.UTF-8/LC_MESSAGES/palma.po (hunk 2 above)
Comment (zuphilip) [inline locale/de_DE.UTF-8/LC_MESSAGES/palma.po]: Yeah, well I would suggest to search and replace all of this links in all languages. I created an issue for that https://github.com/UB-Mannheim/PalMA/issues/41.

**Comment by zuphilip** [comment]
Please have a look at the new version and my comments above.

**Pull request merged**